lib/runtime/procs/read.js

```diff
@@ -9,10 +9,13 @@ var JuttleMoment = require('../../runtime/types/juttle-moment');
 var Promise = require('bluebird');
 
 
-var Read = source.extend({
-    procName: 'read',
+class Read extends source {
+    get procName() {
```

Comment:
You'll have to use `this.procName = 'read';` in constructor instead of this as of https://github.com/juttle/juttle/pull/317, since that does `this.procName += ' ' + adapter.name;` once we have an adapter.